### PR TITLE
completed ticket 5 and refactored my article list to have article cards

### PR DESCRIPTION
**Pull request by 779carney**

### CODE REVIEW

File: src/components/SingleArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../../utils/api";
+
+function SingleArticle(){
+    const[singleArticle, setSingleArticle]= useState({})
+ const {article_id}=useParams()
+ console.log(article_id);
```
Comment (HannahBurks) [inline src/components/SingleArticle.jsx]: Make sure to remove any stray console logs :)

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ useEffect(()=>{
```
Comment (HannahBurks) [inline src/components/SingleArticle.jsx:9]: We also want to add a loading state, so if we are still awaiting the data a loading message can be displayed on the screen

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Nice work Ellie, we just need to add a loading state and message for if the data is loading. Once done, please send a PR tomorrow morning and you should be good to go!

**Review by aloush-dev — APPROVED** [review]
nice looks good,
merge away.

**Pull request merged**